### ISSUE TITLE
`@now/static-build` problem with gatsby

### ISSUE THREAD

**Issue by markozxuu** [vercel/vercel#2156]
**now version**: 15.0.4-canary.6
**build**: @now/static-build

## Steps to reproduce 🐛 

```sh
git clone https://github.com/zeit/now-examples
cd gatsby
yarn install  # dependency installation
now dev # note the error output
```

I'm testing the example of **express-twitter-dreamify** of now-examples

logs

```npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! gatsby-starter-default@1.0.0 build: `gatsby build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the gatsby-starter-default@1.0.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/mark/.npm/_logs/2019-04-13T01_32_39_178Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! gatsby-starter-default@1.0.0 now-build: `npm run build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the gatsby-starter-default@1.0.0 now-build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/mark/.npm/_logs/2019-04-13T01_32_39_378Z-debug.log
> Error! Exited with 1

**Comment by markozxuu** [comment]
This was solved in the version: **15.0.7-canary.8**

Thank you so much @amio @leo @styfle 🙏

**Comment by miyamoto-san** [comment]
I'm still experiencing this issue!

```
npm ERR! code ELIFECYCLE
--
02:42:01 PM | npm ERR! errno 1
02:42:01 PM | npm ERR! gatsby-starter-default@1.0.0 build: `gatsby build`
02:42:01 PM | npm ERR! Exit status 1
02:42:01 PM | npm ERR!
02:42:01 PM | npm ERR! Failed at the gatsby-starter-default@1.0.0 build script.
02:42:01 PM | npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
02:42:01 PM | npm ERR! A complete log of this run can be found in:
02:42:01 PM | npm ERR!     /zeit/.npm/_logs/2020-01-31T13_42_01_450Z-debug.log
02:42:01 PM | Error: Exited with 1
02:42:01 PM | at ChildProcess.<anonymous> (/zeit/7783010c87aa68f0/.build-utils/.builder/node_modules/@now/static-build/dist/index.js:31880:24)
02:42:01 PM | at ChildProcess.emit (events.js:210:5)
02:42:01 PM | at ChildProcess.EventEmitter.emit (domain.js:475:20)
02:42:01 PM | at maybeClose (internal/child_process.js:1021:16)
02:42:01 PM | at Process.ChildProcess._handle.onexit (internal/child_process.js:283:5)
02:42:01 PM | worker exited with code 20 and signal null
```
